### DISCUSSION TITLE
[1.13.0] Your Windows build exceeds the current known support capabilities of your Sandboxie version

### DISCUSSION THREAD

**Discussion by BilleBarrett** [sandboxie-plus/Sandboxie#3635] [General]
why am I seeing  this
22.02.2024 14:12:46.166   
|Time|                    |Message|                                                                                                                                                                               

22.02.2024 14:12:46.166   Your Windows build exceeds the current known support capabilities of your Sandboxie version, Sandboxie will attempt to use the last-known offsets which may cause system instability.   
Thank you

**Comment by bastik-1001** [comment]
What is the exact version of Windows you see this message on? Since 1.13.0 Sandboxie shows that message whenever the Windows build version is newer than what it knows how to handle, while using the last-known offsets. The user is therefore aware why there might be crashes.

**Comment by BilleBarrett** [comment]
It is working OK now, it was a setting I didn’t know about

 

-----------------------------------------------------------------------------------------------------------------


The bridge and the wall are made with the same material

The Bridge joins the people and the Wall divides the people

Choose the Right One

Remote Desktop Services 
(336) 988-0891    
 ***@***.***> ***@***.***

**Comment by DavidXanatos** [comment]
Sandboxie should support Windows up to build 26052 and newer builds should only be available when participating in the Windows Insider program, in which case Sandboxie should try the last known offsets. It seems that the participation in the Insider program is not being detected on your system.

Please specify what Windows build do you have and if you are participating in the Windows Insider program.
